pyproject.toml

```
@@ -32,26 +32,32 @@ redis = "*"
 matplotlib = "*"
 
 [tool.poetry.group.dev.dependencies]
-pyproject-flake8 = "^6.1.0"
+ruff = "*"
 tox = "*"
 moto = "*"
 testfixtures = "*"
 coverage = "*"
-isort = "^5.7"
 yapf = ">=0.32"
 toml = "*"
 lupa = "*"
 fakeredis = "*"
 mypy = "*"
 types-redis = "*"
 
-[tool.flake8]
+[tool.ruff]
+select = [
+  "E4", "E7", "E9", "F", # ruff's default. see https://docs.astral.sh/ruff/settings/#select
```

you can use `extend-select` and don't need to copy ruff's default.

https://docs.astral.sh/ruff/settings/#extend-select

```suggestion
extend-select = [
```

---

fixed in [8b5ee66](https://github.com/m3dev/gokart/pull/345/commits/8b5ee6664aa3a0087168c787493ba6e9ec8abed7)